### ISSUE TITLE
WPF XAML FriendlyName property

### ISSUE THREAD

**Issue by MarceloJGaray** [MarimerLLC/cslaforum#905]
Hello,

Is there a way to get a BO’s FriendlyName property (object's Display attribute) from WPF XAML? I was looking at the CSLA.XAML.PropertyInfo control but it doesn’t look that’s possible.

Version and Platform
CSLA version: 5.1.0
OS: Windows
Platform: .WPF + NET Standard 2.0

**Comment by rockfordlhotka** [comment]
@MarceloJGaray I don't think this ever occurred to anyone (not me anyway) prior to my building the "propertyinfo" equivalent for Blazor.

It should be easy enough to add the equivalent of the [Blazor FriendlyName property](https://github.com/MarimerLLC/csla/blob/master/Source/Csla.Blazor/PropertyInfo.cs#L77).

**Comment by rockfordlhotka** [comment]
@MarceloJGaray I did add a work item to the backlog. I can't promise when I'll get to it, but you are welcome to submit a PR for the change if you'd like to ensure it ends up in the 5.2 release.

**Comment by MarceloJGaray** [comment]
OK, thank you @rockfordlhotka!
